config/prow/release-branch-jobs/1.25.yaml

```diff
@@ -0,0 +1,519 @@
+        preset-dind-enabled: "true"
+        preset-service-account: "true"
+      name: pull-kubernetes-integration
+      path_alias: k8s.io/kubernetes
+      spec:
+        containers:
+          - args:
+              - ./hack/jenkins/test-dockerized.sh
+            command:
+              - runner.sh
+            image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20231113-7213ea5323-1.25
+            name: ""
+            resources:
+              limits:
+                cpu: "6"
+                memory: 15Gi
+              requests:
+                cpu: "6"
+                memory: 15Gi
+            securityContext:
+              privileged: true
+    - always_run: true
+      annotations:
+        description: run with GO_VERSION set to the original go version used for this branch
+      branches:
+        - release-1.25-lts
+      context: pull-kubernetes-integration-go-compatibility
+      decorate: true
+      labels:
+        preset-dind-enabled: "true"
+        preset-service-account: "true"
+      name: pull-kubernetes-integration-go-compatibility
+      path_alias: k8s.io/kubernetes
+      spec:
+        containers:
+          - args:
+              - ./hack/jenkins/test-dockerized.sh
+            command:
+              - runner.sh
+            env:
+              - name: GO_VERSION
+                value: 1.19.1
+            image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20231113-7213ea5323-1.25
+            name: ""
+            resources:
+              limits:
+                cpu: "6"
+                memory: 15Gi
+              requests:
+                cpu: "6"
+                memory: 15Gi
+            securityContext:
+              privileged: true
+    - always_run: true
+      branches:
+        - release-1.25-lts
+      context: pull-kubernetes-e2e-kind
+      decorate: true
+      decoration_config:
+        grace_period: 15m0s
+        timeout: 1h0m0s
+      labels:
+        preset-dind-enabled: "true"
+        preset-kind-volume-mounts: "true"
+      name: pull-kubernetes-e2e-kind
+      path_alias: k8s.io/kubernetes
+      spec:
+        containers:
+          - command:
+              - wrapper.sh
+              - bash
+              - -c
+              - curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" && e2e-k8s.sh
+            env:
+              - name: FOCUS
+                value: .
+              - name: SKIP
+                value: \[Slow\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|PodSecurityPolicy|LoadBalancer|load.balancer|Simple.pod.should.support.exec.through.an.HTTP.proxy|subPath.should.support.existing|NFS|nfs|inline.execution.and.attach|should.be.rejected.when.no.endpoints.exist
+              - name: PARALLEL
+                value: "true"
+            image: gcr.io/k8s-staging-test-infra/krte:v20231012-0288f8bc6c-1.25
+            name: ""
+            resources:
+              limits:
+                cpu: "7"
+                memory: 9000Mi
```

Comment:
9Gi?

Reply:
yeah, that's what they use on k/k test infra.

I'll change it to 

```
            resources:
              limits:
                cpu: "4"
                memory: 8Gi
              requests:
                cpu: "2"
                memory: 4Gi
```

everywhere